DOCUMENTATION.md

```diff
@@ -1,810 +1,780 @@
+|`pending`|GOV.UK Notify received a callback from the provider but the device has not yet responded|Yes|||
+|`sent`|The text message was delivered internationally|Yes|Yes||
+|`failed`|This will return all failure statuses:<br>- `permanent-failure`<br>- `temporary-failure`<br>- `technical-failure`|Yes|Yes||
+|`permanent-failure`|The provider was unable to deliver message, email or phone number does not exist; remove this recipient from your list|Yes|Yes||
+|`temporary-failure`|The provider was unable to deliver message, email inbox was full or phone was turned off; you can try to send the message again|Yes|Yes||
+|`technical-failure`|Email / Text: Notify had a technical failure; you can try to send the message again. <br><br>Letter: Notify had an unexpected error while sending to our printing provider. <br><br>You can leave out this argument to ignore this filter.|Yes|Yes||
+|`accepted`|Notify is printing and posting the letter|||Yes|
+|`received`|The provider has received the letter to deliver|||Yes|
 
-##### `template_type`
+#### templateType (optional)
 
-If omitted all messages are returned. Otherwise you can filter by:
+You can filter by:
 
 * `email`
 * `sms`
 * `letter`
 
-You can omit this argument to ignore the filter.
-
-##### `status`
+#### reference (optional)
 
-You can filter by:
+A unique identifier. This reference identifies a single unique notification or a batch of notifications.
 
-* `sending` - the message is queued to be sent by the provider.
-* `delivered` - the message was successfully delivered.
-* `failed` - this will return all failure statuses `permanent-failure`, `temporary-failure` and `technical-failure`.
-* `permanent-failure` - the provider was unable to deliver message, email or phone number does not exist; remove this recipient from your list.
-* `temporary-failure` - the provider was unable to deliver message, email box was full or the phone was turned off; you can try to send the message again.
-* `technical-failure` - Notify had a technical failure; you can try to send the message again.
+```ruby
+reference: 'your_reference_string'
+```
 
-You can omit this argument to ignore the filter.
+#### older_than (optional)
 
-##### `reference`
+Input the ID of a notification into this argument. If you use this argument, the client returns the next 250 received notifications older than the given ID.
 
-This is the `reference` you gave at the time of sending the notification. The `reference` can be a unique identifier for the notification or an identifier for a batch of notifications.
+```ruby
+older_than: 'e194efd1-c34d-49c9-9915-e4267e01e92e'
+```
 
-You can omit this argument to ignore the filter.
+If you leave out this argument, the client returns the most recent 250 notifications.
 
+The client will only return notifications that are 7 days old or less. If the notification specified in this argument is older than 7 days, the client will return an empty `collection` response.
```

Comment:
I'm not sure if this last sentence needs a bit more context or not. What would happen here is that a  `Notifications::Client::NotificationsCollection` gets returned (as it always does from this method) and it's the `collections` array within this is empty. I guess it's repeated below and there's a fuller explanation there, so I'll let you decide whether to add more detail here or whether it's best to keep the slightly simplified version.